build/build.php

```diff
@@ -261,6 +262,8 @@
 	system("tar -czf $tmp/packages/$targzFilename * > '/dev/null'");
 	message('Build new full tar.bz2 package.', $verbose);
 	system("tar -cjf $tmp/packages/$tarbz2Filename * > '/dev/null'");
+	message('Build new full tar.zst package.', $verbose);
+	system("tar -I \"zstd -19\" -cvf $tmp/packages/$tarzst2Filename * > '/dev/null'");
```

Comment:
```suggestion
	system("tar -I \"zstd -19\" -cvf $tmp/packages/$tarzstFilename * > '/dev/null'");
```
https://github.com/joomla/joomla-cms/pull/42152 verwendet einen höheren Kompressionslevel `-I zstd --ultra -22`
Gib es einen Grund für -**v**erbose?